src/config/ConfigManager.hpp

```diff
@@ -45,6 +46,19 @@ struct SMonitorRule {
     bool                enable10bit = false;
 };
 
+struct SWorkspaceRule {
+    std::string            monitor         = "";
+    std::string            workspaceString = "";
+    std::string            workspaceName   = "";
+    int                    workspaceId     = -1;
+    std::optional<int64_t> gapsIn          = 5;
```

Comment:
since those are optional, shouldn't they be left empty? You're initializing some defaults and then using `value_or` which means it will always use `value` because it's init, no?

Reply:
Ups, correct. Forgot to change it. Should be good now.